nim/app.py

```diff
@@ -21,15 +21,15 @@ def index():
     )
 
 
-@app.route('/update')
+@app.route('/update', methods=['GET', 'POST'])
 def update_game_state():
     state = session['state']
     # player_move = loads('{"pile": 0, "stones": 1}')
     # app.game.update(state, player_move)
     bot_move = app.game.chaos.move(state)
     app.game.update(state, bot_move)
     session['state'] = state
-    return jsonify({'state': state})
+    return jsonify({'playerState': None, 'botState': state})
```

Comment:
I don't get this. The game only has one state :confused:

Reply:
Pushing refactor. No longer applicable.